pkg/server/metrics/metrics.go

```diff
@@ -95,3 +105,13 @@ func (a *ServerMetrics) ConnectionInc(service_method string) {
 func (a *ServerMetrics) ConnectionDec(service_method string) {
 	a.connections.With(prometheus.Labels{"service_method": service_method}).Dec()
 }
+
+// BackendInc increments the number of backend connection.
+func (a *ServerMetrics) BackendInc() {
+	a.backend.With(prometheus.Labels{}).Inc()
```

Comment:
We've gotten in the habit of adding empty labels on our metrics. Not this is needed or a good idea. I believe we can just remove the "With()" call and Inc() directly on the backend.